lib/reporters/verbose.js

```diff
@@ -449,13 +449,6 @@ class VerboseReporter {
 			this.lineWriter.writeLine(colors.error(`${this.previousFailures} previous ${plur('failure', this.previousFailures)} in test files that were not rerun`));
 		}
 
-		if (this.stats.passedKnownFailingTests > 0) {
-			this.lineWriter.writeLine();
-			for (const evt of this.knownFailures) {
-				this.lineWriter.writeLine(colors.error(this.prefixTitle(evt.testFile, evt.title)));
-			}
-		}
-
```

Comment:
I mean this part

Reply:
Yea it seems weird to have it after the stats, and I don't know why we have to draw specific attention to it here.

Reply:
Ok, I see. It seems like everything is working as expected 🤔